lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -1,32 +1,37 @@
 define((require, exports, module) ->
+  getContext = require("ace/ext/papeeria/latex_parsing_context").getContext
   exports.setupPreviewer = (editor, popoverHandler) ->
     katex = null
-    popoverHandler = popoverHandler ? {
+    popoverHandler ?= new class
```

Comment:
Well, you pointed out before, that I should use `@` in `contextHandler` and `selectionHandler`. This would only work with classes. So what should I do?